src/app/FakeLib/WiXHelper.fs

```diff
@@ -588,10 +923,11 @@ let generateWiXScript fileName =
             <Media Id=\"1\" Cabinet=\"media1.cab\" EmbedCab=\"yes\" />
 
             <Directory Id=\"TARGETDIR\" Name=\"SourceDir\">
-              <Directory Id=\"ProgramFilesFolder\" Name=\"ProgramFiles\">
+              <Directory Id=\"@Product.ProgramFilesFolder@\" Name=\"ProgramFiles\">
```

Comment:
Ok, got it, it's only the placeholder for `ProgramFilesFolder` or `ProgramFiles64Folder`, never mind.